### PR TITLE
EC does not choose supported collateral types

### PR DESCRIPTION
**Pull request by dckc**
That choice is reserved to the BLD stakers.

I noticed this at around [10:30 in the recording](https://youtu.be/aLV43VU8eFM?t=634). I'm not sure how to submit a pull request for youtube videos, though :)

### CODE REVIEW

File: lectureNine/lectureNine.md [hunk 1]
```diff
@@ -428,4 +427,4 @@ which is a bounty we completed for Agoric.
 
 ### Reserve
 There's an [ongoing discussion](https://github.com/Agoric/agoric-sdk/discussions/6967) about the `Reserve`, 
-we'll update here once our questions are settled.
\ No newline at end of file
+we'll update here once our questions are settled.
```
Comment (dckc) [inline lectureNine/lectureNine.md]: oops. this whitespace-only change is an artifact of using the web UI for making commits.

**Comment by dckc** [comment]
@anilhelvaci please take a look

**Comment by dckc** [comment]
cc @jeetraut, @hielo777

**Comment by anilhelvaci** [comment]
@dckc Thanks for correcting. That slipped from my attention.

When you think about I should've caught that new supported PSM tokens approved by the BLD stakers as it requires a `core-eval`(I believe it does but correct me if I'm wrong here as well). 

But I'm surprised that new vault collaterals are approved by BLD Stakers. Does that mean there's now two `Electorate`s governing VaultFactory? One for econ committee one for BLD Stakers. But I've to take a look at vaults code to be sure.

**Comment by anilhelvaci** [comment]
But merging this anyway. Thanks again for the correction! @dckc

**Comment by dckc** [comment]
> But I'm surprised that new vault collaterals are approved by BLD Stakers. Does that mean there's now two `Electorate`s governing VaultFactory? One for econ committee one for BLD Stakers.

No; note that adding a new vault collateral is _not_ done by changing a governed parameter. It's done by calling a method on the creatorFacet.

The EC can only change those governed parameters (or make governed API calls). They cannot, in general, call methods on the creatorFacet.

**Comment by anilhelvaci** [comment]
> No; note that adding a new vault collateral is not done by changing a governed parameter. It's done by calling a method on the creatorFacet.

This makes total sense, thanks! So how do you call a method on `creatorFacet`? A core-eval as well?

**Comment by dckc** [comment]
> ... So how do you call a method on `creatorFacet`? A core-eval as well?

Yes; the results of `startInstance` calls, including `creatorFacet` are available to core-eval code; for example:

```js
export const addAssetToVault = async (
  {
    consume: { vaultFactoryKit, agoricNamesAdmin, auctioneerKit },
```
https://github.com/Agoric/agoric-sdk/blob/046eb05f824bddad6e1980c2bfab7b3f57f1e60f/packages/inter-protocol/src/proposals/addAssetToVault.js#L217-L219

The call on the `creatorFacet` is reasonably straightforward:

```js
  const vaultFactoryCreator = E.get(vaultFactoryKit).creatorFacet;
  await E(vaultFactoryCreator).addVaultType(interchainIssuer, oracleBrand, {
```
https://github.com/Agoric/agoric-sdk/blob/046eb05f824bddad6e1980c2bfab7b3f57f1e60f/packages/inter-protocol/src/proposals/addAssetToVault.js#L250-L251

**Pull request merged**